.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: Changelog check
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ opened, synchronize, reopened, labeled, unlabeled ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
Ubuntu 20 is a LTS

Reply:
it's only in preview in github actions and `ubuntu-latest` is still 18.04, so that's the most stable option for now.